packages/language/src/parser/preprocessor-parser.ts

```diff
@@ -1576,7 +1576,15 @@ function stringLiteral(state: ParserState): ast.Literal {
 }
 
 function unpackCharacterValue(literal: string): string {
-  return literal.substring(1, literal.length - 1).replace(/""/g, '"');
+  let length = literal.length;
+  // Remove ending quotes if not escaped
+  if (
+    (literal.endsWith('"') || literal.endsWith("'")) &&
+    literal.charAt(length - 2) !== "\\"
```

Comment:
Ah! Not sure about this one, some languages escape using backslash, some other languages make a double quote `""`. Was not sure if this is a thing in PL/I